### DISCUSSION TITLE
Windows 11 Cygwin64 build

### DISCUSSION THREAD

**Discussion by nothing2see** [cgreen-devs/cgreen#314] [General]
For anyone trying something silly like using cgreen on Windows like me, all you need to do is follow [this guide to install Cygwin](https://www3.ntu.edu.sg/home/ehchua/programming/howto/Cygwin_HowTo.html#cygwin), but also include the `cmake` and `perl` packages. After `make`ing, `make test`ing, and `make install`ing cgreen, include `-L/usr/local/lib -lcgreen` in your linker command.

**Comment by thoni56** [comment]
Thanks for your tip. Cgreen has been building on Cygwin for ages, but it's nice to have someone write up the extra steps.

The problem with Cgreen on "bare" Windows is that there is no straight forward way to port the (very few) Unix-isms, and I suppose the multitude of ways you can build your Windows program. Noone has described how to do that, so there is unfortunately no examples here.
